From: Matthias Klose <doko@debian.org>
Date: Sat, 29 Oct 2022 21:23:31 -0400
Subject: Debian: Degrade argparse gracefully without gettext

python3.X-minimal includes argparse but not gettext. Use a fallback noop
gettext, if it can't be imported.
Forwarded: not-needed
---
 Lib/argparse.py | 11 ++++++++++-
 1 file changed, 10 insertions(+), 1 deletion(-)

diff --git a/Lib/argparse.py b/Lib/argparse.py
index 9be1848..fcdae5c 100644
--- a/Lib/argparse.py
+++ b/Lib/argparse.py
@@ -89,7 +89,16 @@ import os as _os
 import re as _re
 import sys as _sys
 
-from gettext import gettext as _, ngettext
+try:
+    from gettext import gettext as _, ngettext
+except ImportError:
+    def _(message):
+        return message
+    def ngettext(singular,plural,n):
+        if n == 1:
+            return singular
+        else:
+            return plural
 
 SUPPRESS = '==SUPPRESS=='
 
